src/lib/resource/sections/applicant-select-treatments.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
                                 'required_someone-else':
                                     'Enter any other treatment they have received for their mental injury'
                             },
-                            title: 'Other mental health treatment'
+                            title: 'Other treatments'
                         },
```

Comment:
This is a change we need to push back on. This title will be used on the 'Check your answers' page, and work was done a wee while ago to make all these titles something meaningful and not generic. I'd raise this with Julie/Graeme and see if we can get it switched back or to something else.

Reply:
no worries, this was one I noticed on the prototype and discussed with GB 
![image](https://user-images.githubusercontent.com/41323244/141814471-f7780834-6d6d-4aa0-8bd1-ec53167953a7.png)
I will get back to him on this one again.

Reply:
I have changed this back, still waiting on GB getting back to us.

Reply:
GB and Julie are good with changing it back